src/index.html

```diff
@@ -8,9 +8,38 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search__bar--big">
+      <form class="search__bar"
+            action="http://localhost:8080/"
+            data-qa="big">
```

Comment:
Need to fix code style everywhere
```suggestion
      <form
        class="search__bar"
        action="http://localhost:8080/"
        data-qa="big"
      >
```